plugin-jdbc-duckdb/src/main/java/io/kestra/plugin/jdbc/duckdb/Query.java

```diff
@@ -97,6 +98,9 @@ public class Query extends AbstractJdbcQuery implements RunnableTask<Query.Outpu
     @PluginProperty(dynamic = true)
     private String url = DEFAULT_URL;
 
+    @Getter(AccessLevel.NONE)
+    protected transient Path workingDirectory;
```

Comment:
It seems the `wokingDirectory` is only used by the run method, in this case this can be a variable in it instead of a class attribute.